fe/fe-core/src/main/java/org/apache/doris/PaloFe.java

```diff
@@ -173,6 +176,12 @@ public static void start(String dorisHomeDir, String pidDir, String[] args, Star
                 qeService.start();
             }
 
+            ScheduledThreadPoolExecutor refreshTimer = ThreadPoolManager.newDaemonScheduledThreadPool(1,
```

Comment:
This `ScheduledThreadPoolExecutor` should be in `RefreshMgr`.
And there is already a `CatalogMgr` is `Env.java`, why new a `CatalogMgr` here?

Currently, there are 2 main manager:
- RefreshMgr
- CatalogMgr

All refresh method(refresh catalog/db/table/partition) should be in RefreshMgr.
But currently, there is a mistake that "refresh catalog method" is in CatalogMgr.

I think you can do a refactor to move the "refresh catalog method" from CatalogMgr to RefreshMgr.
And this newly added refresh timer thread should be in RefreshMgr too.

Reply:
Please change your IDEA formatter to only format the code you modified. Not changing other code line.

Reply:
1.The reason why I want to create a new `CatalogMgr` is because the `CatalogMgr` needs a timing refresh time, because when the `CatalogMgr` of `Env` is initialized, there is no such timing refresh time, in order to reuse I will subsequently use the `CatalogMgr` of `Env` and assign this refresh time to the` CatalogMg`r of `Env` when it is initialized and constructed
2.Ok,I will  move the "refresh catalog method" from `CatalogMgr` to `RefreshMgr` and putthe refresh timer thread in `RefreshMgr`
3.Ok, I will not change other lines of code